src/adapters/icon_manager.js

```diff
@@ -1,96 +1,27 @@
-import styleIcons from '@qwant/qwant-basic-gl-style/icons.yml';
+import { getPoiIcon } from '@qwant/qwant-basic-gl-style';
 import classnames from 'classnames';
 import { ACTION_BLUE_DARK } from 'src/libs/colors';
 
-const nameToClass = iconName => iconName.match(/^(.*?)-[0-9]{1,2}$/)[1];
-
 export default class IconManager {
   static get({ className, subClassName, type }) {
-
-    // Get the category icon of a PoI
-    if (type === 'poi' || type === 'category') {
-      const icons = styleIcons.mappings;
-
-      // Matching class and subclass
-      let icon = icons.find(iconProperty => {
-        return iconProperty.subclass === subClassName && iconProperty.class === className;
-      });
-
-      // Or: no class and matching subclass
-      if (!icon) {
-        icon = icons.find(iconProperty => {
-          return iconProperty.subclass === subClassName && !iconProperty.class;
-        });
-      }
-
-      // Or: matching class and no subclass
-      if (!icon) {
-        icon = icons.find(iconProperty => {
-          return iconProperty.class === className && !iconProperty.subclass;
-        });
-      }
-
-      if (icon) {
-        const iconName = icon.iconName;
-        const color = icon.color;
-        const iconClass = nameToClass(iconName);
-        return { iconClass, color };
-      }
-
-      return {
-        iconClass: nameToClass(styleIcons.defaultIcon),
-        color: styleIcons.defaultColor,
-      };
-
-    // Get the icon of a location / area that is not a PoI:
-    // Exact address
-    } else if (type === 'house' || type === 'address') {
-      return {
-        iconClass: nameToClass(styleIcons.defaultAddressIcon),
-        color: styleIcons.defaultAddressColor,
-      };
-
-    // Road / street without house number
-    } else if (type === 'street') {
-      return {
-        iconClass: nameToClass(styleIcons.defaultStreetIcon),
-        color: styleIcons.defaultStreetColor,
-      };
-
-    // user geolocation "poi"
-    } else if (type === 'geoloc') {
+    if (type === 'geoloc') {
       return {
         iconClass: 'pin_geoloc',
         color: ACTION_BLUE_DARK,
         icomoon: true,
       };
-
-      // administrative zones (city, area, country)
-    } else {
-      return {
-        iconClass: nameToClass(styleIcons.defaultAdministrativeIcon),
-        color: styleIcons.defaultAdministrativeColor,
-      };
     }
-  }
-}
 
-export function createIcon(iconOptions) {
-  const icon = IconManager.get(iconOptions);
-
-  // Show a white circle instead of marker2 in map markers of PoI that have no class or subclass.
-  if (icon.iconClass === nameToClass(styleIcons.defaultIcon)) {
```

Comment:
This function depended on the YML mapping, but during the simplification I realized we never use it for something other than creating this static marker with no special options.
![Capture d’écran de 2021-01-25 11-10-39](https://user-images.githubusercontent.com/243653/105691837-0431aa80-5efe-11eb-86df-5856e1cf93ee.png)